controllers/servingruntime_controller.go

```diff
@@ -623,3 +676,16 @@ func (r *ServingRuntimeReconciler) clusterServingRuntimeRequests(csr *kserveapi.
 
 	return requests
 }
+
+func (r *ServingRuntimeReconciler) secretRequests(secret *corev1.Secret, namespace string) []reconcile.Request {
```

Comment:
Rename to `storageSecretRequests`? (otherwise a bit generic)

Also don't need a separate `namespace` arg, can just get it from the secret inside this func.